.travis.yml

```
@@ -22,3 +22,4 @@ after_success:
 env:
   global:
   - MIX_ENV=test
+  - CIRCUITS_MIX_ENV=test
```

@th0mas would be good to get this environment variable documented somewhere.
e.g. an in-line comment:
```suggestion
  # GPIO testing, see: https://github.com/elixir-circuits/circuits_gpio#testing
  - CIRCUITS_MIX_ENV=test
```